Tests/SonarScanner.MSBuild.Shim.Test/AdditionalFilesServiceTest.cs

```diff
@@ -354,6 +354,6 @@ public void AdditionalFiles_ExtensionsFound_MultipleProperties_WithAdditionalPar
 
         files.Sources.Should().BeEmpty();
         files.Tests.Should().BeEmpty();
-        logger.AssertWarningLogged($"""The support for multi-language analysis may not function correctly if {param} is set. If this is the case, please explicitly set "sonar.scanner.multiFileAnalysis=false" to disable the multi-language analysis.""");
+        logger.AssertWarningLogged($"""The support for multi-language analysis (scanAll) may not function correctly if {param} is set. If this is the case, please explicitly set "sonar.scanner.scanAll=false" to disable the scanAll analysis.""");
```

Comment:
same here, I would write the last part:

`...to disable the multi-language analysis.`